src/Documentation/Advanced-Concepts/Activation-Garbage-Collection.md

```diff
@@ -64,89 +66,29 @@ Note that `DelayDeactivation` does not 100% guarantee that the grain activation
 
 ## Configuration
 
-### Programmatic Configuration
-
-Default Collection Age Limit (for all grain types) can be set via:
-``` csharp
-  void GlobalConfiguration.Application.SetDefaultCollectionAgeLimit(TimeSpan ageLimit)
-```
-
-For individual grain types the limit can be set via:
-``` csharp
-  void GlobalConfiguration.Application.SetCollectionAgeLimit(Type type, TimeSpan ageLimit)
-```
-
-The limit can also be reset for a grain type, so that the default limit would apply to it, via:
-``` csharp
-  void GlobalConfiguration.Application.ResetCollectionAgeLimitToDefault(Type type)
-```
-
-### XML Configuration (deprecated)
-
-Any length of time in the configuration XML file may use a suffix that specifies a unit of time:
-
-Suffix      |     Unit
-------------- | -------------
-none  | millisecond(s)  
-ms    | millisecond(s)  
-s     | second(s)  
-m     | minute(s)  
-hr    | hour(s)  
-
-
-**Specifying Default Collection Age Limit**
-
-The default collection age limit that applies to all grain types can be customized by adding the OrleansConfiguation/Globals/Application/Defaults/Deactivation element to the OrleansConfiguration.xml file.
-The minimal allowed age limit is 1 minute.
-
-The following example specifies that all activations that have been idle for 10 minutes or more should be considered eligible for deactivation.
-
-``` xml
-<?xml version="1.0" encoding="utf-8"?>
-<OrleansConfiguration xmlns="urn:orleans">
-  <Globals>
-    <Application>
-      <Defaults>
-        <Deactivation AgeLimit="10m"/>
-      </Defaults>
-    </Application>
-  </Globals>
-</OrleansConfiguration>
-```
-
-**Specifying per-Type Age Limits**
-
-Individual grain types may specify a collection age limit that is independent from the global default, using the OrleansConfiguation/Globals/Application/GrainType/Deactivation element. The minimal allowed age limit is 1 minute.
-
-In the following example, activations that have been idle for 10 minutes are eligible for collection, except activations that are instantiations of the MyGrainAssembly.DoNotDeactivateMeOften class, which are not considered collectable unless idle for a full 24 hours:
-
-
-``` xml
-<?xml version="1.0" encoding="utf-8"?>
-<OrleansConfiguration xmlns="urn:orleans">
-  <Globals>
-    <Application>
-      <Defaults>
-        <Deactivation AgeLimit="10m"/>
-      </Defaults>
-      <GrainType Type="MyGrainAssembly.DoNotDeactivateMeOften">
-        <Deactivation AgeLimit="24hr"/>
-      </GrainType>
-    </Application>
-  </Globals>
-</OrleansConfiguration>
-```
-
- Any number of GrainType elements may be specified.
- 
\ No newline at end of file
+Grain garbage collection can be configured using the `GrainCollectionOptions` options:
+
+```csharp
+mySiloHostBuilder.Configure<GrainCollectionOptions>(options =>
+{
+  // Period on which silo will check all inactive grains
+  options.CollectionQuantum = TimeSpan.FromMinutes(1);
```

Comment:
I'm not sure we want to highlight `CollectionQuantum` here. It may confuse some people, as it's more of an implementation detail.